inst/examples/lmBenchmark.R

```diff
@@ -5,7 +5,7 @@
 ## This file is part of RcppEigen.
 
 require("stats", character=TRUE, quietly=TRUE)
-require("rbenchmark", character=TRUE, quietly=TRUE)
+require("microbenchmark", character=TRUE, quietly=TRUE)
```

Comment:
We need to add it to Suggests, and we cannot "just" require it.  Please make the whole block below conditional of microbenchmark being present.

Reply:
Are you sure? `rbenchmark` wasn't `require()`'d previously and the below scripts weren't conditional. Even though this is in `inst/examples`, `R CMD check` doesn't register it as an "example" in the sense of getting checked so this wasn't a problem. I'm happy to add it, just want to make sure you're ok with adding another soft-dependency first.

Reply:
I am not; I just noticed it was not in `DESCRIPTION` but then again `inst/examples/` may not get parsed.

But I also used to use this wrong code idiom of just `require()` which does NOT reliably fail / skip in the absence of the test package. I have changed this.  

So with that, I would still like you to *condition* on `microbenchmark` being present as it is a _conditional dependency_ via `Suggests`.  See what we do here in `tests/doRUnit.R`:

```r
if(require("RUnit", quietly = TRUE)) {
    # ...
```

That is the only correct way for `Suggests`.

Reply:
Ok, done.  

I left `RcppArmadillo` and `RcppGSL` out of "Suggests" unless you want them included also.